mkosi/backend.py

```diff
@@ -561,20 +561,23 @@ class MkosiPrinter:
 
     prefix = "‣ "
 
+    level = 0
+
     @classmethod
     def _print(cls, text: str) -> None:
         cls.out_file.write(text)
 
     @classmethod
     def print_step(cls, text: str) -> None:
+        prefix = cls.prefix + ' ' * cls.level
```

Comment:
might be pretty to draw unicode `╰` character here, to present this is a tree structure?